Brewfile

```diff
@@ -0,0 +1,4 @@
+brew 'postgresql@14'
+brew 'postgis'
```

Comment:
re: https://github.com/18F/identity-reporting-rails/pull/1#discussion_r1424346512, we can remove anything having to do with postgis for now